CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# How to contribute
+
```

Comment:
Yeah. Going to change that and references to jira. 
Is everything else ok?

Just out of curiosity feedhenry is our opensource brand. 
Build farm is aerogear. Can we mention about this project on feedhenry.org website? 
How we can unify this to join communities. I  would see feedhenry as agreggator of all community projects no matter if they are done under aerogear umbrella. 
That's just easy to follow and easy to track. We have now dedicated team that would respond to emals and take care of our community so we may benefit from this.

ping @david-martin